__tests__/app.test.js

```diff
@@ -1,48 +1,89 @@
+const seed = require("../db/seeds/seed");
+const db = require("../db/connection");
+const app = require("../app");
+const data = require("../db/data/test-data");
+const endpoints = require("../endpoints.json");
 
-beforeEach(()=>seed(data))
-afterAll(()=>db.end())
+beforeEach(() => seed(data));
+afterAll(() => db.end());
 
-describe("/api/topics",()=>{
-    test("GET 200: Responds with all topics, each of which with the properties of slug and description",()=>{
-        return request(app)
-        .get("/api/topics")
-        .expect(200)
-        .then(({body})=>{
-            const {topics} = body
-            expect(topics.length).toBe(3)
-            topics.forEach((topic)=>{
-                expect(typeof topic.description).toBe('string')
-                expect(typeof topic.slug).toBe('string')
-            })
-        })
-    })
-})
+describe("/api/topics", () => {
+  test("GET 200: Responds with all topics, each of which with the properties of slug and description", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
+        expect(topics.length).toBe(3);
+        topics.forEach((topic) => {
+          expect(typeof topic.description).toBe("string");
+          expect(typeof topic.slug).toBe("string");
+        });
+      });
+  });
+});
 
-describe("/api",()=>{
-    test("GET 200: Responds with a description of all other endpoints available",()=>{
-        return request(app)
-        .get("/api")
-        .expect(200)
-        .then(({body})=>{
-            expect(body.endpoints).toEqual(endpoints)
-        })
-    })
-})
+describe("/api", () => {
+  test("GET 200: Responds with a description of all other endpoints available", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.endpoints).toEqual(endpoints);
+      });
+  });
+});
 
-describe("/*",()=>{
-    test("ALL 404: Responds with a path not found when an incorrect path is invalid",()=>{
-        return request(app)
-        .get("/invalid")
-        .expect(404)
-        .then(({body})=>{
-            const {msg} = body
-            expect(msg).toBe("Path not found")
-        })
-    })
-})
\ No newline at end of file
+describe("/api/articles/:article_id", () => {
+  test("GET 200: Responds with an object with the articles properties", () => {
+    return request(app)
+      .get("/api/articles/6")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        const testArticle = {
+          article_id: 6,
+          title: "A",
+          topic: "mitch",
+          author: "icellusedkars",
+          body: "Delicious tin of cat food",
+          created_at: "2020-10-18T01:00:00.000Z",
+          votes: 0,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        };
```

Comment:
This test is solid for testing current functionality, however isn't very flexible. If you ever wanted to expand the info from this request and add another key (which we may ask you to do down the line...) this test will fail and need to be updated.
Matchers like `.toMatchObject` or `.objectContaining` are great for this reason. You can check an object contains certain values without limiting values it could contain in the future 🔮